apps/api/src/routers/demo.py

```diff
@@ -15,3 +17,18 @@ async def me(username: Annotated[Union[str, None], Cookie()] = None) -> str:
 async def square(value: Annotated[int, Form()]) -> int:
     """Calculate the square of the value."""
     return value * value
+
+
+@router.post("/user")
+async def get_user(name: Annotated[str, Form()]) -> list[dict[str, Any]]:
```

Comment:
Nitpick [non-blocking since just for demo]: return type could be more specific than `Any`, i.e. `str`.

Reply:
Shouldn't we keep it as `Any` since we could be storing different data types? Also, if we change it to `str` that results in some type errors because the return type for `retrieve` is `list[dict[str, object]]`.